WooCommerce/Classes/ViewRelated/Orders/Order Create/OrderCreationFormViewController.swift

```diff
@@ -0,0 +1,97 @@
+import UIKit
+
+final class OrderCreationFormViewController: UIViewController {
+
+    /// Main TableView.
+    ///
+    private lazy var tableView = UITableView(frame: .zero, style: .grouped)
+    private let viewModel: OrderCreationFormViewModel
+    private let dataSource: OrderCreationFormDataSource
+
+    init() {
+        self.viewModel = OrderCreationFormViewModel()
+        self.dataSource = OrderCreationFormDataSource(viewModel: viewModel)
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        self.viewModel = OrderCreationFormViewModel()
+        self.dataSource = OrderCreationFormDataSource(viewModel: viewModel)
```

Comment:
Is really needed, in this case, to initialize `viewModel` and `dataSource` here?

Reply:
`viewModel` and `dataSource` are non-optional without default value. If we won't init them here in any way - we should block this required initializer and throw `fatalError`.
Any more ideas how to handle 2 inits properly?